src/app/globals.css

```
@@ -193,6 +193,68 @@ a {
+ *   - http://typekit.com/eulas/00000000000000007735fdb8
+ *   - http://typekit.com/eulas/00000000000000007735fdbb
+ * wigwag-bold:
+ *   - http://typekit.com/eulas/00000000000000007735b4e2
+ * wigwag-deluxe:
+ *   - http://typekit.com/eulas/00000000000000007735b4e3
+ *
+ * © 2009-2025 Adobe Systems Incorporated. All Rights Reserved.
+ */
+/*{"last_published":"2025-06-08 01:05:19 UTC"}*/
+
+@import url("https://p.typekit.net/p.css?s=1&k=dym2fcl&ht=tk&f=27844.27874.37515.37516.37521.37522&a=143685011&app=typekit&e=css");
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)